index.html

```diff
@@ -2943,6 +2944,229 @@ <h5><code>unsubscribeallevents</code></h5>
+      The <em>HTTP Webhook profile</em> MAY be used in conjunction with the
+      <a href="#http-baseline-profile">HTTP Baseline Profile</a> in order to
+      provide operations to read and write properties and invoke, query and
+      cancel actions.</p>
+      <p class="rfc2119-assertion" id="http-webhook-profile-protocol-binding-general-2">
+        The <em>HTTP Webhook profile</em> MAY be used as an alternative event mechanism 
+        to the HTTP SEE Profile.</p>
+    <!-- Identifier -->
+    <section id="http-webhook-profile-identifier">
+      <h2>Identifier</h2>
+      <p class="rfc2119-assertion" id="http-webhook-profile-1">
+        In order to denote that a given
+        <a href="https://www.w3.org/TR/wot-architecture/#dfn-thing">Web Thing</a>
+        conforms to the HTTP Webhook Profile, its Thing Description MUST have a
+        <a href="https://w3c.github.io/wot-thing-description/#thing">
+        <code>profile</code></a> member [[wot-thing-description]] with a value
+        of <code>https://www.w3.org/2022/wot/profile/http-webhook/v1</code>.
+      </p>
+    </section>
+
+
+    <section id="http-webhook-profile-protocol-binding">
+      <h2>Protocol Binding</h2>
+      <p>
+        This section defines a protocol binding which describes how a
+        <a href="https://www.w3.org/TR/wot-architecture/#dfn-consumer">Consumer</a>
+        communicates with a
+        <a href="https://www.w3.org/TR/wot-architecture/#dfn-thing">Web Thing</a>
+        [[wot-architecture11]] using Webhook Events [[FIXME: Reference]].
+      </p>
+      <p>
+        <span class="rfc2119-assertion" id="http-webhook-profile-protocol-binding-1">
+        A Consumer or Web Thing conforming to the HTTP Webhook Profile
+        MUST implement this protocol binding.
+        </span>
+      </p>
+
+      <section id="webhook-example">
+        <h3>Webhook-Example</h3>
+  
+        <p>The following sequence diagram illustrates the operation and message flow from subscription to an event via notification on a state change to unsubscription.
+        </p>
+  
+        <figure id="sequence-diagram-webhook">
+          <img src="images/Sequence Diagram - WebHook.png" class="wot-profiles" alt="Webhook example - sequence diagram" />
+          <figcaption>HTTP Webhook Profile - Sequence of operations and notifications</figcaption>
+        </figure>
+  
+  
+        <p>The following example contains a snippet from a TD that illustrates how a Webhook event can be described.</p>
+        <pre class="example">
+        ...
+  
+        {
+            "events": {
+                "fireAlarm": {
+                    "description": "Fire!",
+                    "subscription": {
+                        "type": "object",
+                        "properties": {
+                               ...
+                               listener: uri,
+                               ...
+                            }
+                        }
```

Comment:
I note that this is different to [Example 68](https://w3c.github.io/wot-thing-description/#example-68) in the Thing Description specification which uses object members called `callbackURL` and `subscriptionID` for the subscription payload.

Is the member name "listener" intended to be standardised?

If it is then that needs defining in a normative statement, and `callbackURL` may actually be a better term.

If it's not then how should a Web Thing determine which part of the subscription metadata to use as a callback URL?

What happens if the same consumer requests multiple subscriptions to the same event? Should a subscription ID be returned in the response to the subscription request?

The ellipses could be removed since this is probably all that's needed,